.github/workflows/ci.yml

```diff
@@ -26,7 +26,9 @@ jobs:
                     - 'lowest'
                     - 'highest'
                 remove-dependencies: [ '' ]
-
+                include:
+                  - php-version: '8.2'
+                    symfony-version: '7.0.*'
```

Comment:
As SF7.0 is not applicable for PHP 8.1, added a configuration on top of the matrix, to only run SF7.0 for PHP8.2
@see https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations

Reply:
@mcg-web Could you re-trigger the GH action one more time? :)